app/views/statics/donate.html.erb

```diff
@@ -11,7 +11,7 @@
   <li>Donator+ ($20 or more)
 </ul>
 
-<p>We also have <%= link_to "list of users who donated", users_path(badge: "donor") %> already!</p>
+<p>We also have <%= link_to "list of users who donated", users_path(donor: "") %> already!</p>
```

Comment:
This grammar error has actually been in there for a while now. You can see it on the production server right now. Thanks for pointing this out, I'll add a fix to this pull request.